com.skype.Client.json

```diff
@@ -8,31 +8,24 @@
     "command": "skype",
     "separate-locales": false,
     "finish-args": [
-        /* X11 + XShm access */
```

Comment:
@wjt I don't like removing these comments

Reply:
The problem here is that comments are not valid JSON. This tool uses the standard Python `json` module to parse these files, but to work around the presence of comments like these, it round-trips the data through json-glib to strip comments and other non-standard constructs like multi-line strings.

Reply:
> The problem here is that comments are not valid JSON.

Oh. Totally missed that. Fair enough.